osu.Game/Screens/Play/Player.cs

```diff
@@ -83,7 +83,7 @@ public abstract partial class Player : ScreenWithBeatmapBackground, ISamplePlayb
         /// </summary>
         protected virtual bool PauseOnFocusLost => true;
 
-        public Action<bool> RestartRequested;
+        public Action<bool> PrepareLoaderForRestart;
```

Comment:
What's with the rename? I preferred the old name.

Reply:
I'd say it's because the `PlayerLoader` is no longer responsible for performing the restart, ie. calling `this.MakeCurrent()`. the `Requested` terminology implies that another component is going to make the decision of whether a restart should actually occur.